planner/planner.go

```diff
@@ -18,23 +20,45 @@ func GetPlan(srcDir string) (*Plan, error) {
 	result := &Plan{
 		Packages: []string{},
 	}
-	for _, planner := range PLANNERS {
-		if planner.IsRelevant(srcDir) {
-			plan, err := planner.GetPlan(srcDir)
-			if err != nil {
-				return nil, err
-			}
-			result = MergePlans(result, plan)
+	for _, planner := range getRelevantPlans(srcDir) {
+		plan, err := planner.GetPlan(srcDir)
+		if err != nil {
+			return nil, err
 		}
+		result = MergePlans(result, plan)
 	}
 	return result, nil
 }
 
 func HasPlan(srcDir string) bool {
+	return len(getRelevantPlans(srcDir)) > 0
+}
+
+func IsBuildable(srcDir string) (bool, error) {
+	buildables := []Planner{}
+	for _, planner := range getRelevantPlans(srcDir) {
+		if plan, err := planner.GetPlan(srcDir); err != nil {
+			return false, err
+		} else if !plan.Buildable() {
+			if plan.buildHint != "" {
+				return false, usererr.New(plan.buildHint)
+			}
+			return false, usererr.New("Unable to build project")
+		}
+		buildables = append(buildables, planner)
+	}
+	if len(buildables) > 1 {
+		return false, usererr.New("Multiple buildable plans found: %v", buildables)
```

Comment:
"Multiple buildables" -> I worry the user won't understand what that is.

I don't love that the user has no way of recovering here (through no fault of their own). Have you seen this happen in practice?